frontend/src/pages/programs.tsx

```diff
@@ -1,11 +1,20 @@
-import { useRedirectTo404IfNotAdmin, useRedirectToLoginIfNotSignedIn } from "@/hooks/redirect";
+import { ProgramData } from "../components/ProgramForm/types";
+import ProgramFormButton from "../components/ProgramFormButton";
+import sampleProgramData from "../sampleProgramData.json";
+
+//import { useRedirectTo404IfNotAdmin, useRedirectToLoginIfNotSignedIn } from "@/hooks/redirect";
 
 export default function Programs() {
-  useRedirectToLoginIfNotSignedIn();
-  useRedirectTo404IfNotAdmin();
+  //useRedirectToLoginIfNotSignedIn();
+  //useRedirectTo404IfNotAdmin();
   return (
-    <main>
-      <h1>PIA Programs Page!</h1>
-    </main>
+    <div>
+      <div className="w-40">
+        <ProgramFormButton type="add" />{" "}
+      </div>
+      <div className="w-40">
+        <ProgramFormButton type="edit" data={sampleProgramData as ProgramData} />{" "}
```

Comment:
Before we merge it in lets remove edit program button. Don't want to add additional files to main like sampleProgramData.json